datastore/exporters/nmdb-export-port-list/nmdb-export-port-list.cpp

```diff
@@ -160,8 +167,7 @@ class Tool : public nmdt::AbstractExportTool
       }
       else {
         // Open and parse the port list file.
-        auto& nmfm {nmcu::FileManager::getInstance()};
-        sfs::path const portListPath {nmfm.getConfPath()/"port-list.conf"};
+        sfs::path const portListPath {this->getConfPath()};
```

Comment:
If you change the prior, this needs to be fixed to get the value of the configuration option.  For example:
```suggestion
        sfs::path const portListPath {opts.getValue("config-path")};
```